### PR TITLE
CLOUD-3270 - There is no environment variable to configure the initial metaspace size

### PR DESCRIPTION
**Pull request by luck3y**
https://issues.jboss.org/browse/CLOUD-3270
Signed-off-by: Ken Wills <kwills@redhat.com>

Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [ ] Pull Request title is properly formatted: `[CLOUD-XYA] Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket
- [ ] Attached commits represent units of work and are properly formatted
- [ ] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [ ] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

File: jboss/container/java/jvm/bash/artifacts/opt/jboss/container/java/jvm/java-default-options [hunk 1]
```diff
@@ -144,6 +145,7 @@ gc_config() {
        "-XX:MaxHeapFreeRatio=${maxHeapFreeRatio} "\
        "-XX:GCTimeRatio=${timeRatio} "\
        "-XX:AdaptiveSizePolicyWeight=${adaptiveSizePolicyWeight} "\
+       "-XX:MetaspaceSize=${metaSpaceSize}m"
```
Comment (yersan) [inline jboss/container/java/jvm/bash/artifacts/opt/jboss/container/java/jvm/java-default-options]: It looks like there is a missing space and "\\ at on this line at the end of this line.

Thi change throws `/opt/run-java/java-default-options: line 149: -XX:MaxMetaspaceSize=192m: command not found` error in the console

**Comment by jmtd** [comment]
This looks good but it  would be nice to have some tests for it… but I notice `gc.feature` got ejected from the repo and moved to https://github.com/jboss-container-images/openjdk/blob/develop/tests/features/java/gc.feature

How about, tests are not a blocker for merging this but some kind of gentleman's agreement that we can cook up some for *that* repo in the near future? :-)

**Comment by yersan** [comment]
Hello @jmtd, maybe I'm going to add more confusion to this conversation, it's completely unintentional, are not the tests for this issue available as PR over jboss-eap-modules [here](https://github.com/jboss-container-images/jboss-eap-modules/pull/138)?

**Comment by jmtd** [comment]
Ah no that's really helpful, thanks — I don't mind where they are so long as I can run them :-) And I can copy those into the openjdk image tests too, and worry about re-unification at a later date.

**Pull request merged**